### ISSUE TITLE
PythonExamples install fails for 3.0.0-rc1/rc2 on Raspbian 64-bit with TAP 9.19.0 

### ISSUE THREAD

**Issue by alan-copeland-keysight** [opentap/OpenTap.Python#51]
The paths seem kind of funky. Installed TAP with the Linux install script. Added $HOME/bin to $PATH. TAP is installed in $HOME/.tap
```
lvltuser@raspberrypi:~ $ tap package install PythonExamples --version -rc
Updates available for:
 - PythonExamples: 3.0.0-rc.1+67ab614d -> 3.0.0-rc.2+fa0f1492
 - OpenTAP: 9.19.0+22538968 -> 9.19.1+2d14a151
Downloaded 'PythonExamples' to '/home/lvltuser/.local/share/OpenTap/PackageCache/PythonExamples.3.0.0-rc.2+fa0f1492.Windows,Linux,MacOS.TapPackage'. [25.5 ms]
Installing to /home/lvltuser/.tap
Uninstalled PythonExamples version 3.0.0-rc.1+67ab614d. [182 ms]
Installing /home/lvltuser/.local/share/OpenTap/PackageCache/PythonExamples.3.0.0-rc.2+fa0f1492.Windows,Linux,MacOS.TapPackage
Installing requirements from Packages/PythonExamples/requirements.txt
ERROR: Could not open requirements file: [Errno 2] No such file or directory: '/home/lvltuser/Packages/PythonExamples/requirements.txt'
Package action OpenTap.Python.PythonInstallAction failed [20.2 s]
Failed to install packages

```

I don't understand why its looking for requirements.txt from here?
`/home/lvltuser/Packages/PythonExamples/requirements.txt`

**Comment by rmadsen-ks** [comment]
It seems like we expect the working directory to be the .tap folder, but actually it is where ever the process is started from. Let's use a better way of finding the requirements.txt file.

**Comment by rmadsen-ks** [comment]
Fixed by #53
